packages/flame/test/effects/color_effect_test.dart

```diff
@@ -18,7 +18,11 @@ void main() {
       game.update(0);
       expect(
         component.paint.colorFilter.toString(),
-        equals('ColorFilter.mode(Color(0x00f44336), BlendMode.srcATop)'),
+        equals(
```

Comment:
This is super sus - not sure this is right approach ofc but values seem rounded

Reply:
Interesting that we compare it with the string, maybe that's the easiest way to check it.
Let's keep it like this for now.